draft-looker-jwm.xml

```diff
@@ -28,9 +28,9 @@
     <date day="21" month="December" year="2019"/>
 
     <abstract>
-      <t>JSON Web Message (JWM) is a flexible way of describing an application-level message encoded in JSON that is suitable 
-      for transfer over a variety of transport protocols. JWMs through the application of JSON Web Signature (JWS) and JSON Web 
-      Encryption (JWE) can be digitally signed, integrity protected and or achieve confidentiality via encryption.</t>
+      <t>JSON Web Message (JWM) is a flexible way to encode application-level messages in JSON
+      for transfer over a variety of transport protocols. By using JWMs in conjunction with JSON Web Signature (JWS) and JSON Web
+      Encryption (JWE), messages can achieve confidentiality, integrity, and non-repudiation.</t>
```

Comment:
Perhaps this last sentence

"...., messages can be digitally signed, integrity protected and/or achieve properties such as confidentiality and non-repudiable authentication via encryption"

Reply:
I like the way @tplooker suggestion splits up the properties based on the underlying cryptography used.

Reply:
I took Tobias's suggestion, but split it into two sentences for readability.